ftw/theming/resources/scss/elements/form.scss

```diff
@@ -19,8 +18,7 @@ $minimum-textarea-lines: 4 !default;
     IE or FF do not display the height of the inputfield correctly so
     we have to set the height manually and applying the content-box model.
```

Comment:
This comment is no longer correct, since you have removed the `box-sizing` below, right? Could you remove it?